vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpCookieSteps.java

```diff
@@ -89,4 +92,40 @@ public void saveHttpCookieIntoVariable(String cookieName, Set<VariableScope> sco
             }
         }
     }
+
+    /**
+     * Change cookie value.
+     * If present several cookies with the same name will be changed value for all of them,
+     *
+     * @param cookieName     name of cookie
+     * @param newCookieValue value to set
+     */
+    @When("I change value of all HTTP cookies with name `$cookieName` to `$newCookieValue`")
+    public void changeHttpCookieValue(String cookieName, String newCookieValue)
+    {
+        List<Cookie> cookies = findCookiesBy(FILTER_BY_NAME, cookieName);
+        if (softAssert.assertThat(String.format(NUMBER_OF_COOKIES_WITH_NAME, cookieName), cookies.size(),
+                greaterThan(0)))
```

Comment:
move the assertion to a new method, that's going to be a method with single line of code, but it will improve overall code readability